frontend/packages/ceph-storage-plugin/integration-tests/views/subscriptionFlow.view.ts

```diff
@@ -0,0 +1,199 @@
+
+export const OCS_OP = 'Openshift Container Storage Operator';
+
+export const NS = 'openshift-storage';
+
+export const operatorsPage = async () => {
+  await sideNavView.clickNavLink(['Operators', 'Installed Operators']);
+  await browser.wait(until.and(crudView.untilNoLoadersPresent));
+};
+
+export const ocsOperator = $('.co-clusterserviceversion-logo__name__clusterserviceversion');
+export const ocsOperatorStatus = $('.co-clusterserviceversion-row__status');
+
+const ocsLink = (elem, catalogSource) =>
+  $(`a[data-test="${elem}-${catalogSource}-openshift-marketplace"]`);
+
+export const goToOperatorHub = async () => {
+  await sideNavView.clickNavLink(['Operators', 'OperatorHub']);
+  await browser.wait(until.and(crudView.untilNoLoadersPresent));
+};
+
+const searchInputOperatorHub = $('input[placeholder="Filter by keyword..."]');
+
+export const searchInOperatorHub = async (searchParam) => {
+  await browser.wait(until.visibilityOf(searchInputOperatorHub));
+  await searchInputOperatorHub.sendKeys(searchParam);
+  const ocs = await ocsLink('ocs-operator', 'ocs-catalogsource');
+  await browser.wait(until.visibilityOf(ocs));
+  return ocs;
+};
+
+export const installOperator = $('.pf-m-primary');
+export const uninstallOperator = $('.pf-m-secondary');
+export const subscribeButton = $('.pf-m-primary');
+
+export const closeOperatorInstall = $('.close');
+
+export const dropdownForNamespace = $('#dropdown-selectbox');
+export const selectNamespace = (namespace: string) => $(`#${namespace}-Project-link`);
+export const status = $('.co-icon-and-text');
+
+// Subscription page for ocs installation mode
+export const customNamespaceRadio = $('input[value="OwnNamespace"]');
+
+// Operators page
+export const storageClusterLink = $('article:nth-child(10) a');
+
+export enum InstallPlan {
+  Automatic = 'Automatic',
+  Manual = 'Manual',
+}
+
+export class InstallCluster {
+  namespace: string;
+
+  constructor(namespace) {
+    this.namespace = namespace;
+  }
+
+  async installOperator() {
+    await browser.wait(until.and(crudView.untilNoLoadersPresent), 100000);
+    await customNamespaceRadio.click();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(dropdownForNamespace));
+    await dropdownForNamespace.click();
+    const nsTag = selectNamespace(this.namespace);
+    await nsTag.click();
+    await subscribeButton.click();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+  }
+}
+
+export const btnCreate = $('.pf-m-primary');
+const selectAllBtn = $('[aria-label="Select all rows"]');
+
+export const selectWorkerRows = async () => {
+  const isAllSeleted = await selectAllBtn.isSelected();
+  if (isAllSeleted === true) await selectAllBtn.click();
+  const workerAz = [];
+  await browser.wait(until.presenceOf($('[data-label="Role"]')), 10000);
+  const table = $('.pf-c-table.pf-m-compact tbody');
+  const rows = table.$$('tr');
+  rows.each(async (row, index) => {
```

Comment:
I'm afraid that this implementation has a potential to fail on `NoStaleElementException`. It would be more safe to fetch the text of the whole table, parse it to find out which rows are worker nodes and then loop over the worker rows to select the checkboxes.
@spadgett  WDYT?